lib/dradis/plugins/nexpose/gem_version.rb

```diff
@@ -9,7 +9,7 @@ def self.gem_version
       module VERSION
         MAJOR = 4
         MINOR = 8
-        TINY = 1
+        TINY = 2
```

Comment:
We should bump the MINOR version here. Otherwise, we'll have to add a `No changes` entry for v4.9.0.